cmd/query/app/flags_test.go

```diff
@@ -60,6 +84,28 @@ func TestQueryBuilderBadHeadersFlags(t *testing.T) {
 	assert.Nil(t, qOpts.AdditionalHeaders)
 }
 
+func TestQueryOptionsWithFlags_CheckHostPort(t *testing.T) {
```

Comment:
These tests for different syntax of host-port string would better to into `./ports/ports_test.go`. This file tests interaction with CLI flags.

Reply:
moved to ./ports/ports_test.go